test/gtest/shp/transform.cpp

```diff
@@ -0,0 +1,65 @@
+// SPDX-FileCopyrightText: Intel Corporation
+//
+// SPDX-License-Identifier: BSD-3-Clause
+#include "shp-tests.hpp"
+
+namespace drtesting {
+std::optional<bool> expect_fast_transform_path;
+}
+
+#include "dr/shp/algorithms/transform.hpp"
```

Comment:
```suggestion
```

Already included in `shp-tests.hpp`

Reply:
It is not included (only view is).
And I need `expect_fast_transform_path` to be defined before including `dr/shp/algorithms/transform.hpp` to allow extra checks.

I'd like to split shp-tests.hpp and create some header which will contain only test common stuff without including all shp code. Then use this header here, include only needed stuff.

Reply:
shp.hpp should include everything. User programs do not have individual includes.